mono/metadata/domain.c

```diff
@@ -984,6 +984,7 @@ mono_domain_set_internal_with_options (MonoDomain *domain, gboolean migrate_exce
 
 	SET_APPDOMAIN (domain);
 	SET_APPCONTEXT (domain->default_context);
+    mono_gc_wbarrier_generic_nostore (&domain->default_context);	
```

Comment:
Whitespace

Reply:
No fair I commented first that I was going to fix it!